src/crc32c.c

```diff
@@ -400,8 +400,6 @@ int unittest_crc32c (void) {
         uint32_t crc;
         const char *how;
 
-        crc32c_global_init();
```

Comment:
doesn't seem right to remove this?

Reply:
https://github.com/edenhill/librdkafka/commit/3237ed9020f477edd480d4dac07b91f9ca72f3ff

Reply:
oh, i see you do a global init.